src/Parser.js

```diff
@@ -561,6 +561,11 @@ class Parser {
                     throw new ParseError(
                         "Can't use function '" + func + "' in text mode",
                         baseGroup.token);
+                } else if (this.mode === "math" &&
+                    funcData.allowedInMath === false) {
+                    throw new ParseError(
+                        "Can't use function '" + func + "' in math mode",
```

Comment:
Indeed. (And thanks for teaching me ES interpolation strings in the first place!) You can tell I was copy/pasting from above. 😄